pypika/queries.py

```diff
@@ -1055,6 +1069,12 @@ def _from_sql(self, with_namespace=False, **kwargs):
               for clause in self._from
         ))
 
+    def _force_index_sql(self, **kwargs):
+        return ' FORCE INDEX({indexes})'.format(indexes=','.join(
+            index.get_sql(with_alias=True, subquery=True, **kwargs)
```

Comment:
just curious why you're explicitly adding `with_alias=True, subquery=True` here? could result in an exception if **kwargs already contains those values.